cypress/page/upload.page.ts

```diff
@@ -0,0 +1,31 @@
+class UploadPage {
+    private uploadToPageURL: string;
+    private selectFile: string;
+    private uploadBtn: string;
+    private msgconfirmUpload: string;
+    private uploadedFile: string;
+  
+    constructor() {
+      this.uploadToPageURL= "https://the-internet.herokuapp.com/upload";
+      this.selectFile = "#file-upload";
+      this.uploadBtn = "#file-submit";
+      this.msgconfirmUpload = "#content";
+      this.uploadedFile = "#uploaded-files";
+    }
+  
+    public visitPage(): void {
+      cy.visit(this.uploadToPageURL);
+    }
+  
+    public uploadFile(file: string): void {
+      cy.get(this.selectFile).attachFile(file);
+      cy.get(this.uploadBtn).click();
+    }
+  
+    public msgConfirmation(fileName: string, msg:string): void {
+        cy.get(this.msgconfirmUpload).should("have.text", msg);
+      cy.get(this.uploadedFile).should("contain", fileName);
+    }
+  }
+  
+  export {UploadPage};
```

Comment:
EOF

Reply:
still see the EOF warning
![image](https://user-images.githubusercontent.com/10209180/182659691-a155c7d5-db46-4002-88f0-2e662c608224.png)